makefile.shade

```diff
@@ -8,4 +8,4 @@ k-standard-goals
 custom-goals
 
 #initialize if='Directory.Exists("src")'
-  exec program='dotnet' commandline='run -p ../../tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode Http/FrameHeaders.Generated.cs Http/Frame.Generated.cs' workingdir='${kestrelSrcDir}'
```

Comment:
This is changing the paths that the generated code is written to, not paths related to the generator itself.